### PR TITLE
Binding autosave to page unload so it only happens once.

### PR DESCRIPTION
**Pull request by eliilek**
Per Stephanie I am making a pull request! Hooray! That said someone who's not on windows should probably test this.

Resolves: #262

### CODE REVIEW

**Comment by eliilek** [comment]
I screwed up a thing hang on, sorry.

File: app/views/characters/edit.slim [hunk 1]
```diff
@@ -38,10 +38,14 @@ javascript:
...
+		
+		$(window).bind("beforeunload", function(){
 			formAutosave();
 		});
-		$('form input[type="hidden"], form input[type="checkbox"], form input[type="radio"]').on('change', function () {
-			formAutosave();
-		});
-	});
\ No newline at end of file
+		//$('form input, form textarea, form select').on('blur', function () {
```
Comment (sublimemarch) [inline app/views/characters/edit.slim]: Remove commented out code.

**Review by sublimemarch — CHANGES_REQUESTED** [review]

**Comment by sublimemarch** [comment]
Nope, this doesn't solve the issue.

screen shot 2016-09-19 at 10 40 44 pm

**Comment by carlydotwebsite** [comment]
I think part of the continuing issue might be that submitting the form also triggers unload, so the elements are definitely submitted at least twice. Might need to unbind it on the submit event.  (That doesn't explain why steph is still getting it a huge number of times—after I pulled this branch it only duplicates the new stuff once—but might help, anyway.)

**Comment by sublimemarch** [comment]
It works! Hurray!

**Review by sublimemarch — APPROVED** [review]

**Pull request merged**